tests/integration/mock/CodeActAgent/test_edits/prompt_002.log

```diff
@@ -235,25 +227,8 @@ The server is running on port 5000 with PID 126. You can access the list of numb
 --- END OF EXAMPLE ---
 
 
-NOW, LET'S START!
-
-----------
-
-Fix typos in bad.txt. Do not ask me for confirmation at any point.
-
-ENVIRONMENT REMINDER: You have 9 turns left to complete the task.
```

Comment:
Looks like a regression. This is missing in the new prompt.

Reply:
Did some digging here--I think this is a feature not a bug :)

This is the ENVIRONMENT REMINDER from a _previous_ prompt, which got hard-coded into the message history. Now there will only be a single ENVIRONMENT REMINDER per prompt

Reply:
(You can see the new ENVIRONMENT REMINDER for 8 turns below this)